test-scripts/local.conf.appendix

```diff
@@ -0,0 +1,12 @@
+# Enable testing
+INHERIT += " \
+    testimage \
+    testexport \
+"
+
+# All test suites to run
+TEST_SUITES = " \
+    ping \
+    ssh \
+    auto \
+"
```

Comment:
This file can be removed right, there's another version with the same name that I think is the one used.

Reply:
Oops. A merge mistake.

Reply:
My idea here was to have one file for the test specific env. Maybe it's overkill? What do you think @tobsan (or others)?

Reply:
A more specific one is nice but I guess only one should be used?

Reply:
There is configuration needed for the tests, but not for anything else. That was shy it felt most reasonable for me to have a new conf file for the tests.